### ISSUE TITLE
bitter preference 

### ISSUE THREAD

**Issue by slaulederkind** [obophenotype/mammalian-phenotype-ontology#3194]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

1.  bitter preference 
2.  low bitter preference 

>2. Synonyms (e.g., absent spleen)

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

1.  subjects demonstrate a difference for bitter tasting substances compared to other offered substance
2.  subjects demonstrate an avoidance of bitter tasting substances compared to other offered substance

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

PMID:16923148

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

1.  abnormal food preference (MP:0001432)
2.  bitter preference  (new term)

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

RGD:gthayman

**Comment by anna-anagnostop** [comment]
Added two new terms as siblings under abnormal food preference (MP:0001432):

1. bitter preference MP:0031056
DEF: subjects demonstrate predilection for bitter tasting substances over other offered substances

2. low bitter preference MP:0031056
DEF: subjects demonstrate a low predilection for bitter tasting substances over other offered substances

Also, made sweet preference (MP:0011976) and low sweet preference (MP:0012347) siblings. 
[low sweet preference (MP:0012347) was previously a child of sweet preference (MP:0011976)]
